scripts/update-app.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+OA_BRANCH=$1
+if [[ $OA_BRANCH = "develop" ]]; then
+    CD_LOCATION="/home/dev/oa"
+elif [[ $OA_BRANCH = "master" ]]; then
+    CD_LOCATION="/var/www/oa"
+else
+    echo "ERROR: Invalid branch specified"
+    exit
+fi
+
+cd $CD_LOCATION
+git reset --hard && git clean -fd && git checkout $OA_BRANCH && git pull
```

Comment:
```suggestion
git reset --hard && git clean -fd && git checkout $OA_BRANCH && git pull
echo "$OA_BRANCH pulled at `date`" >> /var/log/git_logs
```
It might be a good idea to keep track of when the last pull was.

Reply:
Since this script would run on the system crontab every X amount of minutes or so, this log would fill up quite fast. Though I could make this change with the `>` operator instead of `>>`, this way `git_logs` would only show the last pull time instead of the 1000's of other pulls that may accumulate over the next year.

Reply:
Yeah if you're running it regularly than you don't have to log it, didn't know it was gonna run on crontab. 
Looks good to merge. 👍